web_sheet_full_width_selective/models/model.py

```diff
@@ -0,0 +1,59 @@
+# coding: utf-8
```

Comment:
Hello, I won't go against the OCA rules, but let me tell you I agree with @bealdav and I would like this topic be debated again. With this rule, some modules like in our BR localization will soon have 50 files instead of 10 making them totally impossible to grasp... Just like Java projects...